blog/_posts/2016-07-14-ola-from-l1ghtsab3r.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introducing Team l1ghtsab3r India
+layout: post
+created_at: Thur Jul 14 2016
+permalink: blog/introducing-team-l1ghtsab3r-india
+current: blog
+author: Team l1ghtsab3r
+categories:
+- student-posts
+---
+
+
+### About Us <img src="../img/blog/2016/l1ghtsab3r-saber-purple.png" style="height: 25px; width: 60px">
+
+Hi! We are [Soumya Sharma](https://twitter.com/soummyaah) and [Srishti Sengupta](https://twitter.com/SenguptaSrishti) from New Delhi, India. This is our first time participating in an open source program. We are very excited to be associated with it and working for our project. Our project is VOC and is a part of the PyBeeWare community.
+
+### Meet our team <img src="../img/blog/2016/l1ghtsab3r-saber-pink.png" style="height: 25px; width: 60px">
+
+<center><img src="../img/blog/2016/l1ghtsab3r-team.png" alt="The l1ghtsab3r team!" style="width: 600px;"></center>
```

Comment:
the images don't need a `<center>` or a `<style>` tag; this is all handled internally by the css. adding a fixed width will break the responsive view.